tests/unit/Tag/TagTitleTest.php

```diff
@@ -63,6 +63,8 @@ function () {
                 Tag::setTitle($value);
 
                 expect(Tag::getTitle())->equals("<title>{$value}</title>" . PHP_EOL);
+                
+                expect(Tag::getTitle(false))->equals("{$value}" . PHP_EOL);
```

Comment:
You need to rebase it know onto 3.0.x, but this will be a long process. Easier will be just force checkout 3.0.x and make changes again and create new PR.